public/less/take-a-stand.less

```diff
@@ -163,75 +192,104 @@
       }
     }
     .pol-pic-link {
-      font-size: 2rem;
+      // font-size: 2rem;
       font-weight: bold;
+      text-align: center;
       .text-font();
       > img {
         border-radius: 50%;
         border: 2px solid @blue;
         height: 18rem;
-        @media (min-width: @medium-bp) and (max-width: @large-bp) {
-          height: 15rem;
+        @media (max-width: @small-bp) {
+          width: 20rem;
+        }
+        @media (min-width: @small-bp) and (max-width: @medium-bp) {
+          height: 17rem;
+        }
+        @media (min-width: @medium-bp) {
+          height: 14rem;
         }
       }
     }
     .pol-name {
+      margin: auto;
+      text-align: center;
       margin-top: 1rem;
       min-height: 6rem; // uneven heights create irregular floating
       font-family: Oswald;
+      @media (max-width: @small-bp) {
+        font-size: 2rem;
+      }
+      @media (min-width: @small-bp) and (max-width: @medium-bp) {
+        font-size: 2rem;
+      }
+      @media (min-width: @medium-bp) {
+        font-size: 1.5rem;
+      }
     }
   }
 
   // ************************* STEP 4 *************************
   .step4 {
-    .twibbot-overlay-container {
-      // margin: auto;
-    }
-    .twibbot-overlay {
-      margin-bottom: 1em;
-      position: relative;
+    text-align: center;
+    .twibbon-overlay {
       margin: auto;
-      &, &:after, > img {
-        height: 100%;
-        width: 100%;
-        max-width: 40rem;
-      }
-      &:after {
-        content: '';
-        background: url('/photos/twibbot.png');
-        z-index: 0;
-        background-size: contain;
-        background-repeat: no-repeat;
-        position: absolute;
-        top: 0;
-        left: 0;
+      margin-bottom: 1em;
+      background: url('../photos/andy.jpg') no-repeat;
```

Comment:
haha, I thought you might not want to be on the page….but the picture is better, more centered for the overlay.  

What is the purpose of the `:after` pseudo class….I couldn’t make it work with the different pic sizes

On Feb 9, 2016, at 2:51 PM, Andy Ernst <notifications@github.com> wrote:

> In public/less/take-a-stand.less:
> 
> > -      &, &:after, > img {
> > -        height: 100%;
> > -        width: 100%;
> > -        max-width: 40rem;
> > -      }
> > -      &:after {
> > -        content: '';
> > -        background: url('/photos/twibbot.png');
> > -        z-index: 0;
> > -        background-size: contain;
> > -        background-repeat: no-repeat;
> > -        position: absolute;
> > -        top: 0;
> > -        left: 0;
> > +      margin-bottom: 1em;
> > +      background: url('../photos/andy.jpg') no-repeat;
> No one wants to see me. Change it back
> 
> —
> Reply to this email directly or view it on GitHub.
>